src/Web/Handler.hs

```diff
@@ -147,6 +200,161 @@ getTheory idx = do
     liftIO $ withMVar (theoryVar yesod) $ return. M.lookup idx
 
 
+getLemmaPlaintext :: Int -> TheoryPath -> Handler String
+getLemmaPlaintext nr path = do
+    let lname = case path of 
+            (TheoryEdit n) -> n 
+            _ -> "this should never be visible"
```

Comment:
If this should really be unreachable, I think it would be better to return `error "unreachable"`.